source/VexiiRiscv/Introduction/index.rst

```diff
@@ -1,20 +1,19 @@
 Introduction
 ============
 
-
-About VexiiRiscv
-------------------------------
-
-VexiiRiscv is a from scratch second iteration of VexRiscv, with the following goals : 
-
-- RISCV 32/64 bits IMAFDC
-- Could start around as small as VexRiscv, but could scale further in performance
-- Optional late-alu
-- Optional multi issue
-- Optional multi threading
-- Providing a cleaner implementation, getting ride of the technical debt, especially the frontend
-- Proper branch prediction
-- ...
+VexiiRiscv is the next generation of VexRiscv, enabling things which are not possible with the current VexRiscv framework.  The new VexiiRiscv framework: 
+
+- Supports both the 32 bit and the 64 bit Risc-V Instruction sets. 
```

Comment:
We losed the definition of what extentions are supported IMAFDC

Hmm overall, i didn't meant to replace the first paragram / bullet points with a direct VexRiscv comparison, but instead to add the comparison in the same chapter. Something short and direct.